.github/workflows/ci.yml

```diff
@@ -2,6 +2,17 @@ name: Continuous integration
 
 on:
   pull_request:
+    paths-ignore:
+      - '.cookiecutter/*'
+      - '.editorconfig'
+      - '.gitignore'
+      - '.mailmap'
+      - '.pyup.yml'
+      - 'AUTHORS'
+      - 'HACKING.md'
+      - 'LICENSE'
+      - 'NOTICE'
+      - 'docker-compose.yml'
```

Comment:
Note that in h's case we actually do want to run `ci.yml` on changes to `README.rst` and `docs/*` because of the `tox -e checkdocs` command (it checks that the Sphinx documentation at https://h.readthedocs.io/ builds successfully) but for other projects these are just markdown files and we'll be able to ignore them.